build-configs/integreat/theme/colors.ts

```diff
@@ -1,8 +1,10 @@
 import { ColorsType, commonLightColors } from '../../common/theme/colors'
 
 const themeColor = '#fbda16'
+const themeColorLight = 'rgba(251, 218, 22, 0.5)'
```

Comment:
You could also use hex notation here: `#fbda1680`
See e.g. https://rgbacolorpicker.com/rgba-to-hex

Regarding the used colors: I am fine with this as this was proposed by a designer, but if you want we can also wait for additional expertise from our new designer starting on Tuesday :)

Reply:
@klinzo It is for sure the best option to ask the new designer which colors we should chose here

Reply:
I would say we merge this and create a new issue :)